experimental/auth-logic/endorsement-release/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"strings"
+)
+
+// This program accepts multiple auth logic files as input by passing them
+// into an array variable. To allow a []string to be used in flag.var, it must
+// implement flag.Value, which includes String() and Set(). And to be able to
+// define these, it must be given a type name.
+type stringArray []string
+
+func (someStringArray *stringArray) String() string {
+	return strings.Join(*someStringArray, ", ")
+}
+
+func (someStringArray *stringArray) Set(value string) error {
+	*someStringArray = append(*someStringArray, value)
+	return nil
+}
+
+func main() {
+
+	var authLogicInputs stringArray
+
+	appName := flag.String("app_name", "", "set name of application to be released")
+	provenanceFilePath := flag.String("provenance", "", "set path of provenance file")
+	outputAuthLogicFilePath := flag.String("auth_logic_out", *appName+"_endorsement_release_policy.auth_logic", "set path for generated output authorization logic verification code")
+	flag.Var(&authLogicInputs, "auth_logic_inputs", "set one or more auth logic input files; to provide multiple inputs, call the parameter more than once with each input, for example, `--auth_logic_inputs file1 --auth_logic_inputs file2`")
+
+	flag.Parse()
+
+	// Part of the code for building a project using provenance
+	// files changes the working directory. This binary needs to keep
```

Comment:
Really? Which part changes the working directory?

Reply:
This comment is just referring to #60. The comment is just directly copied from the clinet-verification code.

Reply:
Is this part of the code even related to #60? Does the code here call `FetchSourcesFromRepo`? I think it should not.

Reply:
I think it does indirectly... let me check more thoroughly later...